docs/operation/cluster_monitoring/index.rst

```diff
@@ -0,0 +1,12 @@
+Cluster Monitoring
+==================
+
+This section covers the MetalK8s monitoring and alerting stack operations. It
+also describes the metric resources that are automatically monitored once
+MetalK8s is deployed (for example Prometheus), and the list of pre-configured
+alerting and recording rules.
```

Comment:
Unsure what "metric resources" mean, and why Prometheus is an example of that.

I'd suggest:

```suggestion
This section covers the MetalK8s monitoring and alerting stack operations. It
also describes the metrics monitored using Prometheus, with the list of
pre-configured alerting and recording rules.
```